content/cookbook/wallets/check-publickey.md

```diff
@@ -11,7 +11,7 @@ have a private key associated with them. You can check this by looking to see if
 the public key lies on the ed25519 curve. Only public keys that lie on the curve
 can be controlled by users with wallets.
 
-```javascript file="check-public-key.ts"
+```javascript file=/code/cookbook/wallets/check-public-key.ts#L1-L13
```

Comment:
should this include the other lines since they are showing how to validate the keys?
```suggestion
```javascript file=/code/cookbook/wallets/check-public-key.ts#L1-L19
```